src/Espalexa.h

```diff
@@ -159,12 +161,27 @@ class Espalexa {
     if (static_cast<uint8_t>(dev->getType()) > 1)
       sprintf(buf_cm,PSTR("\",\"colormode\":\"%s"), modeString(dev->getColorMode()));
     
-    sprintf_P(buf, PSTR("{\"state\":{\"on\":%s,\"bri\":%u%s%s,\"alert\":\"none%s\",\"mode\":\"homeautomation\",\"reachable\":true},"
-                   "\"type\":\"%s\",\"name\":\"%s\",\"modelid\":\"%s\",\"manufacturername\":\"Philips\",\"productname\":\"E%u"
-                   "\",\"uniqueid\":\"%s\",\"swversion\":\"espalexa-2.7.0\"}")
-                   
-    , (dev->getValue())?"true":"false", dev->getLastValue()-1, buf_col, buf_ct, buf_cm, typeString(dev->getType()),
-    dev->getName().c_str(), modelidString(dev->getType()), static_cast<uint8_t>(dev->getType()), buf_lightid);
+
+    if (static_cast<uint8_t>(dev->getType()) == 0)
+    {
+       // On/Off
+        sprintf_P(buf, PSTR("{\"state\":{\"on\":%s,\"alert\":\"none%\",\"reachable\":true},"
+                       "\"type\":\"%s\",\"name\":\"%s\",\"modelid\":\"%s\",\"manufacturername\":\"Philips\",\"uniqueid\":\"%s\",\"swversion\":\"espalexa-2.7.0\"}")
+                      
+        , (dev->getValue())?"true":"false", typeString(dev->getType()),
+        dev->getName().c_str(), modelidString(dev->getType()), buf_lightid);
+        Serial.println(buf);
```

Comment:
This `Serial.println()` seems to be a debug leftover.

Note: I would love to see this PR merged. I tested the changes myself and they worked nice, thanks for doing this!